apps/builder/app/builder/features/style-panel/style-source/style-source.tsx

```diff
@@ -110,21 +111,19 @@ export type ItemSource = "token" | "componentToken" | "tag" | "local";
 
 type EditableTextProps = {
   label: string;
-  isEditable: boolean;
   isEditing: boolean;
   onChangeEditing: (isEditing: boolean) => void;
   onChangeValue: (value: string) => void;
 };
 
 const EditableText = ({
   label,
-  isEditable,
   isEditing,
   onChangeEditing,
   onChangeValue,
 }: EditableTextProps) => {
   const { ref, handlers } = useContentEditable({
-    isEditable,
+    isEditable: false,
```

Comment:
did you mean true?

Reply:
good catch, also weird why renaming still works

Reply:
ah it is for editing via double click, fixed now